shared/domain/src/commonMain/kotlin/network/bisq/mobile/client/websocket/subscription/ModificationType.kt

```diff
@@ -0,0 +1,10 @@
+package network.bisq.mobile.client.websocket.subscription
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+enum class ModificationType{
```

Comment:
can we provide a better name for this one? Like DataUpdateRequestType or similar?

Reply:
In JavaFx ListChangeListeners they call it Change (though its a bit diff. design). This is certainly not a good fit here.
It also should not be restricted to a certain collection type. But I guess it will only apply to collections as add/remove only makes sense for those. So maybe CollectionMutationTypr or CollectionModificatinType?

Asked ChatGPT for a good recommendation and it came up with ModificationType as the best choice (i did not mention that term in my query). So maybe its not that bad ;-). I would need to change the bisq2 side as well...

```
Query: What is a good term for the ADD or REMOVE or REPLACE operations on a collection? I pack those operation types in an enum and look for a good enum name .


Reply:
A good term for an enum representing operations like ADD, REMOVE, and REPLACE on collections could be:
Suggestions

    ModificationType (Most Common and Intuitive):
        Describes the purpose clearly: it represents types of modifications you can make to a collection.

ChangeType:

    Concise and easy to understand.
    Reflects the idea of changes applied to the collection.

OperationType:

    Emphasizes the fact that these are specific operations on the collection.

CollectionAction:

    Focuses on the actions performed on a collection.


DeltaType:

    Inspired by the concept of "deltas" (changes) in data structures.
    Works well if your enum is used in contexts where the changes to the collection are tracked over time.

Recommendation

    ModificationType is the most intuitive for general-purpose use cases, as it directly conveys the intent of modifying a collection.
    If your application focuses on change tracking or versioning, ChangeType or DeltaType might be more contextually appropriate.
```

Reply:
Loved the `CollectionModificationType` idea

Reply:
Will consider to rename it, but as I am in the middle of other work, I prefer to do it in a next larger Websocket PR both both Bisq 2 and mobile.

Reply:
sounds good Henrik